packages/gauge/test/gauge_ios_test.dart

```diff
@@ -8,30 +8,18 @@ import 'dart:io';
 
 import 'package:test/test.dart';
 
-import 'package:measure/commands/base.dart';
+import 'package:gauge/commands/base.dart';
 
 void main() {
-  const String measureRootPath = '.';
+  final String gaugeRootPath = Directory.current.absolute.path;
   final String resourcesRootPath = BaseCommand.defaultResourcesRoot;
-  BaseCommand.doEnsureResources(resourcesRootPath, isVerbose: true);
-
-  test('help works', () {
```

Comment:
Why remove this test?

Reply:
Sorry, the initial patch was full of irrelevant changes due to local branching error... I've now cleaned up the patch, and added a unit test.